docs/joss_paper/paper.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'PyUnfold: A Python package for iterative unfolding'
+tags:
+  - Python
+  - unfolding
+  - estimation
+  - statistics
+  - deconvolution
+  - inverse problems
+authors:
+  - name: James Bourbeau
+    orcid: 0000-0003-2164-7789
+    affiliation: "1"
+    email: james.bourbeau@icecube.wisc.edu
+  - name: Zigfried Hampel-Arias
+    orcid: 0000-0003-0253-9117
+    affiliation: "1, 2"
+    email: zhampel@wipac.wisc.edu
+affiliations:
+ - name: University of Wisconsin - Madison, Wisconsin, USA
+   index: 1
+ - name: IIHE, Université Libre de Bruxelles, Bruxelles, Belgium
+   index: 2
+date: 05 May 2018
+bibliography: paper.bib
+---
+
+# Summary
+
+PyUnfold is an extensible framework for the deconvolution of discrete probability 
+distributions via the iterative unfolding method described in [@agostini]. 
+This method falls into the general class of inverse problems, and is especially powerful 
+when no analytic form of the inverse function is explicitly available, instead requiring
+an estimate (e.g. via a finite amount of simulation) of the response function.
+Given that measured data comprise a finite sample, PyUnfold also implements the uncertainty
+contributions stated in [@adye2].
+
+
+The primary purpose of PyUnfold is to provide an unfolding toolkit for members of all 
```

Comment:
What do you think about switching the order of this paragraph and the paragraph that follows? I think it might flow better. If this switch is made, then the first two sections will talk about the unfolding method in general. Then move to software-related topics.